godal.go

```diff
@@ -2052,6 +2075,15 @@ func (sr *SpatialRef) AutoIdentifyEPSG() error {
 	return nil
 }
 
+// Validate SRS tokens.
+func (sr *SpatialRef) Validate() error {
+	ogrerr := C.OSRValidate(sr.handle)
```

Comment:
OSRValidate must be wrapped inside a new function to add to godal.cpp in order to handle errors correctly